test/test_workflow_Run2016B.py

```diff
@@ -63,22 +66,19 @@
 #         os.remove(fname)
 
 # check drifts and create IOV
-#filename = 'BS_result_PiltBeam2021_byIOV/beamspot_Express_PilotBeam2021_.txt'
 count = 0
 for irun, ibs in allBS.items():
     if irun not in specialRuns:
-      pairs = splitByDrift(ibs, slopes = True)    
+      pairs = splitByDrift(ibs, slopes = True, maxLumi = 20)    
     else:
-      pairs = splitByDrift(ibs, slopes = True, maxLumi = 1)    
+      pairs = splitByDrift(ibs, slopes = True)    
     for p in pairs:
         myrange = set(range(p[0], p[1] + 1)) & set(ibs.keys())
         bs_list = [ibs[i] for i in sorted(list(myrange))]
         aveBeamSpot = averageBeamSpot(bs_list)
-#        fname = filename.replace('_.txt', '_'+str(count)+'.txt')
```

Comment:
this should also be restored, because it's used to create 1 txt file per each IOV, which is needed for creating the db file.

Reply:
I added a comment to mark this ;)